docs/examples/201/key-vault-secret-create/main.bicep

```diff
@@ -39,7 +52,13 @@ param skuName string = 'standard'
 @secure()
 @description('Specifies all secrets {"secretName":"","secretValue":""} wrapped in a secure object.')
 param secretsObject object = {
-  secrets: []
+  secrets: [
```

Comment:
let's just uncomment this. I'd rather this example work by default and create a superfluous secret then have it fail